README.md

```diff
@@ -1 +1,66 @@
-# Open edX LTI Store (openedx-ltistore)
+# Openedx LTI Store
+
+A plugin for openedx-platform to store LTI configurations centrally. This allows course creators to add [LTI components](https://edx.readthedocs.io/projects/edx-partner-course-staff/en/latest/exercises_tools/lti_component.html) without having to enter the details like secrets, keys and URLs everytime the component block is created.
+
+## Development
+
+The development instructions are written with the [Open edX Devstack](https://edx.readthedocs.io/projects/open-edx-devstack/en/latest/index.html) as reference.
+
+1. Clone the repostiory to `<devstack_root>/src/` directory
+   ```
+   cd <devstack_root>/src/
+   git clone git@github.com:open-craft/openedx-ltistore.git
+   ```
+2. Install the plugin inside the container
+   ```
+   cd <devstack_root>/devstack/
+   make studio-shell
+   pip install -e /edx/src/openedx-ltistore
+   ```
+3. Run the migrations:
+   ```
+   make studio-shell
+   python manage.py cms migrate lti_store
+   exit
+   ```
+4. Restart the cms service `make studio-restart`
```

Comment:
@tecoholic This also needs to be installed in the LMS env, otherwise it'll fail to launch when published